src/components/_basic.less

```diff
@@ -63,7 +63,8 @@ a.sbtn {
 .yellow-btn {
   .colorRule(@yellow);
 
-  &:not(.fill-color-btn) {
+  &:not(.fill-color-btn),
```

Comment:
After testing it, I think this should be &:not(.fill-color-btn):not(.ripple-btn). Sorry for the mistake!

Reply:
@shahednasser Did the changes, please review them.